index.d.ts

```diff
@@ -1104,7 +1104,7 @@ declare module 'evergreen-ui' {
   }
 
   export interface MenuGroupProps extends Omit<PaneProps, 'title'> {
-    title?: JSX.Element
+    title?: JSX.Element | string
```

Comment:
I'd probably say that this is not quite right. The more "correct" way would be to turn this title, into a `ReactElement | ReactText` (or simply `ReactChild`).

Ultimately it depends on how it's used. Is it simply `<p>{title}</p>`, or is there a `crateElement(title, null, null)`?

Reply:
I think `React.ReactNode` is the most accurate type for `title` given how it's used: https://github.com/segmentio/evergreen/blob/master/src/menu/src/MenuGroup.js#L25

Reply:
@tomitrescak did you try `React.ReactNode` ?

Reply:
Ok, I updated the PR